ncs/Kconfig

```diff
@@ -4,6 +4,27 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+# The following nine KConfigs are maintained for backward compatibility with the sdk-nrf-next repository.
+# They are slated for removal once sdk-nrf-next is no longer in use.
+config SOC_NRF54H20_ENGA_CPUAPP
```

Comment:
Have we ever used this Kconfig?

Reply:
Used  [here](https://github.com/nrfconnect/suit-generator/pull/103/files#diff-4ad152b400bb31725d53b60cfff80cf8ed20105a8d24983374043f55ae906bdfR50) only in NCS context.

Reply:
ENGA KConfigs removed since not available anymore in the sdk-nrf